### PR TITLE
chore(code_quality): add error check

### PR DESCRIPTION
**Pull request by hmbig2**
**What this PR does / why we need it**:
add error check: these unchecked errors can be a critical bug

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

### CODE REVIEW

File: huaweicloud/services/dew/resource_huaweicloud_kps_keypair.go [hunk 1]
```diff
@@ -314,12 +314,19 @@ func parseEncodeValue(b []byte, err error) (string, error) {
 	return *rst, nil
 }
 
-func writeToPemFile(path, privateKey string) error {
-	var err error
+func writeToPemFile(path, privateKey string) (err error) {
```
Comment (niuzhenguo) [inline huaweicloud/services/dew/resource_huaweicloud_kps_keypair.go]: suggest to move writeToPemFile to util/common, it's duplicate with compute_keypair's

Comment (hmbig2) [reply huaweicloud/services/dew/resource_huaweicloud_kps_keypair.go:317]: OK, fix it in the next PR

Comment (hmbig2) [reply huaweicloud/services/dew/resource_huaweicloud_kps_keypair.go]: done

**Pull request merged**